File: src/java/WebNinja/src/test/java/webninja/selenuim/ProjectForm.java
package webninja.selenuim;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;

public class ProjectForm {
    private final WebDriver webDriver;

    public ProjectForm(WebDriver webDriver) {
        this.webDriver = webDriver;
    }

    public void setName(String name) {
        webDriver.findElement(By.name("project_data[Title]")).sendKeys(name);
    }

    public void setDescription(String description) {
        webDriver.findElement(By.name("project_data[Description]")).sendKeys(description);
    }

}

File: src/dotnet/WebNinja/WebNinja/selenium/ProjectForm.cs
using OpenQA.Selenium;

namespace WebNinja.selenium
{
    public class ProjectForm : FormObject
    {
        public ProjectForm(IWebDriver driver) : base(driver)
        {
        }

        public string Name
        {
            set
            {
                Driver.FindElement(By.Name("project_data[Title]")).SendKeys(value);
            }
        }

        public string Description
        {
            set
            {
                Driver.FindElement(By.Name("project_data[Description]")).SendKeys(value);
            }
        }
    }
}
